### DISCUSSION TITLE
How are the Solana network fees set by the Phantom wallet?

### DISCUSSION THREAD

**Discussion by hyetigran** [phantom/docs#203] [General]
I noticed that Phantom will set a network fee at $0.01 (currently) if a priority instruction (example below) is NOT provided in the transaction.

How is this network fee of $0.01 set, is it dynamic? If an fee instruction is set that is below Phantom's implicit amount, how likely is it for the transaction to fail?

```
const addPriorityFeeInstruction = ComputeBudgetProgram.setComputeUnitPrice({
    microLamports: fee,
  });
 ```

**Comment by bfriel** [comment]
Hi @hyetigran , we currently set priority fees on all transactions unless a dapp already specifies a `computeUnitBudget` or `computeUnitLimit` instruction at the time of signing ([more info](https://docs.phantom.app/developer-powertools/solana-priority-fees)). We price these dynamically based on the accounts in question, and will always include a minimum priority fee. We aim for 100% inclusion and will adjust our pricing algorithm based on market conditions like the JUP airdrop event.

One thing to note is that under the current network conditions, priority fees are not deterministic. We expect that to improve with Solana's upcoming `1.18` release.
